src/core/linalg/tests/4C_linalg_tensor_operations_test.cpp

```diff
@@ -217,6 +217,132 @@ namespace
     EXPECT_EQ(A_ddot_B, 112.0);
   }
 
+  TYPED_TEST(TensorOperationsTest, ThreeTens_ddot_Mat)
```

Comment:
You are not supposed to use underscores in test name as per the Gtest docs. Also, why not `ThreeTensor`? I read this as three "10"s first.